### PR TITLE
Add send mail

### PR DESCRIPTION
**Pull request by JjuliaP**
## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-7)

## Code reviewers

- [x] @dmytrorevak
- [x] @IvanAndrukh
- [ ] @JjuliaP
- [x] @makarman
- [x] @noobhunters
- [x] @Silent777
- [x] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by makarman — DISMISSED** [review]

**Review by makarman — APPROVED** [review]

File: eventually/templates/emails/update_password.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<h2>Your password has changed</h2>
+<hr>
+<p>Hi <i>{{ first_name }}</i>,</p>
+<p>We wanted to let you know that your password was changed.</p>
```
Comment (lhalam) [inline eventually/templates/emails/update_password.html:4]: add endline

**Review by redmyt — APPROVED** [review]

File: eventually/utils/send_mail.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+"""This is send mail method"""
+from smtplib import SMTPRecipientsRefused
+from django.conf import settings
```
Comment (lhalam) [inline eventually/utils/send_mail.py:3]: `from django.conf import settings ` -> `from django.conf.settings import  DEFAULT_FROM_EMAIL `

File: eventually/utils/send_mail.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    :param template: string with title of template
+    :type template: string
+
+    :param ctx: set with parameters for html template
+    :type ctx: set
+
+    :return:
+    """
+    from_email = settings.DEFAULT_FROM_EMAIL
```
Comment (lhalam) [inline eventually/utils/send_mail.py:29]: remove  this line

File: eventually/utils/send_mail.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+"""This is send mail method"""
+from smtplib import SMTPRecipientsRefused
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+
+
+def send_email(subject, message, recipient_list, template, ctx):
```
Comment (lhalam) [inline eventually/utils/send_mail.py:8]: add argument of the function  `from_email = DEFAULT_FROM_EMAIL`

**Pull request merged**